libs/ballot-interpreter/src/hmpb-rust/timing_marks.rs

```diff
@@ -715,49 +845,115 @@ pub fn find_complete_timing_marks_from_partial_timing_marks(
     partial_timing_marks: &Partial,
     debug: &ImageDebugWriter,
 ) -> FindCompleteTimingMarksResult {
-    let top_line = &partial_timing_marks.top_rects;
-    let bottom_line = &partial_timing_marks.bottom_rects;
-    let left_line = &partial_timing_marks.left_rects;
-    let right_line = &partial_timing_marks.right_rects;
+    const ALLOWED_TIMING_MARK_INSET_PERCENTAGE: UnitIntervalValue = 0.15;
```

Comment:
Here's what a 15% inset looks like:
![image](https://github.com/votingworks/vxsuite/assets/1938/6cd928f4-8225-4987-a2e5-45cf04dc29e7)

I tried going lower than this (10%), but one of our regression tests began failing (`test_inferred_missing_metadata_from_one_side`). Admittedly, that test image is *very* skewed and has a large inset.

Reply:
I measured it, and the max timing mark border rotation is 1.83º in this fixture:
<img width="431" alt="image" src="https://github.com/votingworks/vxsuite/assets/1938/bf9669d9-b929-40e4-8137-2d07ab8e70d7">
The most I saw in the set of images @arsalansufi gave me from the latest hardware was about 1.3º.

Reply:
Given your description of that one regression test, I would suggest killing it and reducing to 10%.

Reply:
Here's the debug image for the fixture with a dark red rect showing the inset:
![image](https://github.com/votingworks/vxsuite/assets/1938/07b6d994-f10a-40a2-9f3a-ffc386b66230)

I'm somewhat inclined to leave it at 15%, but if we want to try 10% I can run it against the latest images from @arsalansufi and the m15 backups.

Reply:
I'm shocked that image works at all! Impressive.

I would be very comfortable, in fact I'd encourage us to not attempt to scan this image, and if that means we take less risk in our extrapolation, then that's a win.

Reply:
On second though and after looking at this debug image for several of @arsalansufi's latest images, I want to try 10%.

Reply:
Alright, here's the worst rotation from the latest images (top rotation 1.49º):
![2479d422-714f-4d6e-b79a-45571f5439d1-front_debug_partial_timing_marks](https://github.com/votingworks/vxsuite/assets/1938/9c6e43e5-7824-42b7-9d0d-6bd9f06c3bbb)

The timing marks are still fairly comfortably within the 10% inset, so let's go with 10%.